Dockerfile

```diff
@@ -6,6 +6,8 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/lib
 # Needed for cudf.concat(), avoids "OSError: library nvvm not found"
 ENV NUMBAPRO_NVVM=/usr/local/cuda/nvvm/lib64/libnvvm.so
 ENV NUMBAPRO_LIBDEVICE=/usr/local/cuda/nvvm/libdevice/
+ENV DEBIAN_FRONTEND=noninteractive
+ARG CUDA_REL=9.2
```

Comment:
You can add `ARG CUDA_REL=${CUDA_VERSION}` before the FROM line and then redeclare `ARG CUDA_REL` after the from line to have it consume from the single `CUDA_VERSION` build arg.

Reply:
See `CUDA_SHORT_VERSION` here: https://github.com/rapidsai/gpuci-build-environment/blob/master/Dockerfile#L7

Reply:
Ah, very nice. Will update